### ISSUE TITLE
Autocomplete with search

### ISSUE THREAD

**Issue by saileereddy** [maxivak/bootstrap3_autocomplete_input#6]
I want to search fields of users table i.e name & email and display email in autocomplete list

Can it be done?

**Comment by saileereddy** [comment]
Hi,

I have added code to work for search query but I dont want id field as I am searching for email field in users table.

Then it gives me, error as undefined property replace for hidden field id

**Comment by maxivak** [comment]
use these options:
{ :column_name=>'name', :display_value => 'email', :full_model=>true}

It will search by name and display email.

If you want custom search conditions like search by name and email then you need override controller action.

**Comment by saileereddy** [comment]
Yes, I have added custom action to override controller but still it was giving me error for  -
undefined property replace for hidden field id

```
 Cannot read property 'replace' of undefined
```

```
function autocomplete_get_hidden_field(t){var a=t.attr("data-field-id").replace(/(:|\.|\[|\])/g,"\\$1")
```

Controller action -

```
def autocomplete_user_name_email
    users = User.where('name ilike ? or email ilike ?', "#{params[:q]}%", "#{params[:q]}%")
    users_email_ids_array = []
    users.each do |user|
      users_email_ids_array << [user.id, user.email]
    end
    render json: users_email_ids_array.to_json
  end
```

**Comment by saileereddy** [comment]
Output of my search query -

```
[["d3b4bcbf-5709-4643-836a-00d5c864962e","sailee.reddy@icicletech.com"],["46c7a72d-4ca5-407f-91f0-ca8c634c60ad","admin@icicletech.com"]]
```

**Comment by saileereddy** [comment]
Input field -

```
<%= f.input email, label: false, as: :autocomplete, source_query: autocomplete_user_name_email_events_url, field_id: false, input_html: { placeholder: 'Search by name or email' } %>
```

I dont want hidden field for id as this is input tag for email field

**Comment by maxivak** [comment]
It was a bug.

It is fixed in a new version '0.1.9'. Try it.
